sunpy/net/dataretriever/sources/goes.py

```diff
@@ -92,25 +96,24 @@ def _get_url_for_timerange(self, timerange, **kwargs):
         """
         # find out which satellite and datatype to query from the query times
         base_url = 'https://umbra.nascom.nasa.gov/goes/fits/'
-        start_time = datetime.datetime.combine(timerange.start.date(),
-                                               datetime.datetime.min.time())
+        start_time = apTime(datetime.datetime.combine(timerange.start.datetime.date(),
+                                                      datetime.datetime.min.time()))
```

Comment:
Forgive the use of `datetime` here?

Reply:
I think that it might be better to go through strings here :man_facepalming: